content/en/docs/instrumentation/net/manual.md

```diff
@@ -210,6 +211,53 @@ In the preceding example, `childOperation` is ended because the scope of the
 `using` block is explicitly defined, rather than scoped to `DoWork` itself like
 `parentOperation`.
 
+## Creating independent Activities
+
+The previous examples showed how to create Activities that follow a nested
+heirarchy. In some cases, you'll want to create independent Activities that are
+siblings of the same root rather than being nested.
+
+```csharp
+public static void DoWork()
+{
+    using var currentActivity = MyActivitySource.StartActivity("MyActivity");
+
+    using var sibling1 = MyActivitySource.StartActivity(
+        "Sibling1",
+        ActivityKind.Internal, 
+        parentContext: currentActivity is null ? default : currentActivity.Context);
```

Comment:
This could arguably also be done as a check before creating the other activities.

....although this is really just akwardness with the system.diagnostics API.

Reply:
Why would one want to do this? It seems to be complicated and error prone.

I think the common scenario should be:

```csharp
using (var parent = DemoSource.StartActivity("Parent"))
{
    using (var child1 = DemoSource.StartActivity("Child1"))
    {
    }
    using (var child2 = DemoSource.StartActivity("Child2"))
    {
    }
    using (var child3 = DemoSource.StartActivity("Child3"))
    {
    }
}
```

Reply:
Huh, not sure why I didn't think to do it that way. Yes, that's a better code sample. The one downside is it means you can't use `using var` without the bigger API call, but I think that's going to be very unlikely for independent/sibling spans that track independent work in the same scope.

Reply:
I actually used `using var` for the parent in the updated sample, let me know what you think. Happy to also have that one nest as well; I just generally prefer to use more modern C# features in code samples if they fit the scenario.